### ISSUE TITLE
Failure to initialise after updating to OctoPi 0.18.0 & OctoPrint 1.8.3     

### ISSUE THREAD

**Issue by ruralaccent** [UnchartedBull/OctoDash#3197]
I updated my three Raspberry Pi4 octopi installations today to Octoprint 1.8.3.

On the two installations where OctoDash wasn't connected to the printer, it failed to pass the initialisation screen when the system was restarted. 

On the installation where OctoDash was up and running and had been connected to the printer, it was displayed 'port is dead', and when I rebooted that installation, it then got stuck on the initialisation screen like the other two.

Octoprint functions as per normal and I can print just fine via OctoPrint, but OctoDash is now broken and won't ever finish initialising.

I all cases I used the prompt in octoprint to update to the latest version. 

browser.user_agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36
connectivity.connection_check: 1.1.1.1:53
connectivity.connection_ok: True
connectivity.enabled: True
connectivity.online: True
connectivity.resolution_check: octoprint.org
connectivity.resolution_ok: True
env.hardware.cores: 4
env.hardware.freq: 1500.0
env.hardware.ram: 3960369152
env.os.bits: 32
env.os.id: linux
env.os.platform: linux
env.plugins.pi_support.model: Raspberry Pi 4 Model B Rev 1.2
env.plugins.pi_support.octopi_version: 0.17.0
env.plugins.pi_support.throttle_check_enabled: True
env.plugins.pi_support.throttle_check_functional: True
env.plugins.pi_support.throttle_state: 0x0
env.python.pip: 22.0.4
env.python.version: 3.7.3
env.python.virtualenv: True
octoprint.last_safe_mode.date: unknown
octoprint.last_safe_mode.reason: unknown
octoprint.safe_mode: False
octoprint.version: 1.8.3
printer.firmware: Prusa-Firmware 3.10.1 based on Marlin
systeminfo.generated: 2022-09-21T19:46:37Z
systeminfo.generator: zipapi

**Comment by Al3601** [comment]
same issue after upgrading from 0.180 to 1.8.3

**Comment by ruralaccent** [comment]
Following advice from Charlie over on the Octoprint forum, this issue relates to an API bug in 1.8.3 that prevents the global api working correctly. To resolve the issue (and future proof OctoDash as the global api is depreciated), follow these steps:

1. Set up an application api key for OctoDash in OctoPrint,
2. Update the OctoDash config file via SSH with that new application api key,
3. Reboot the system and OctoDash should initialise just fine.

**Comment by cp2004** [comment]
The global API key usage should be fixed in the next bugfix release (1.8.4 I think), however now is a good time to realise that eventually the global API key will go away entirely and to switch to using an application key, or the API key specific to your user.

**Comment by jneilliii** [comment]
Probably the same problem, you're using a global api key and not an application key in your OctoDash settings? You can edit that using the command `nano /home/pi/.config/octodash/config.json` via SSH.

**Comment by lluisd** [comment]
> Probably the same problem, you're using a global api key and not an application key in your OctoDash settings? You can edit that using the command `nano /home/pi/.config/octodash/config.json` via SSH.

yes it was easy to solve it by doing that, but I didn't expect a breaking change by jumping to 1.8.3.

**Comment by cp2004** [comment]
It wasn't a deliberate breaking change, and is now fixed with 1.8.4.

The global API key *will* be going away so it's a good reminder to not use it 😉

**Comment by army11b1980** [comment]
I just did a fresh install and now I'm stuck on the welcome screen. It's like the touch is not responding anymore. I will try to do this solution.

**Comment by queengooborg** [comment]
Touchscreen issue can be fixed via https://github.com/UnchartedBull/OctoDash/wiki/Troubleshooting#touchscreen-is-not-responding
